src/views/registration-js/attendees/fields/url.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * This template renders a Single Ticket content
+ * composed by Title and Description currently
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/registration-js/attendees/fields/url.php
+ *
+ * @version TBD
+ *
+ * @see Tribe__Tickets_Plus__Meta__Field__Url
+ *
+ */
+$required   = isset( $field->required ) && 'on' === $field->required ? true : false;
```

Comment:
* :no_entry_sign: There must be exactly one blank line after the file comment
* :no_entry_sign: Additional blank lines found at end of doc comment

<sub>hash: e1c9735bcfe06f9e021bfa0d47f0d4ec</sub>